src/main/java/com/example/toikprojekt2022/service/UserService.java

```diff
@@ -44,9 +44,19 @@ public User registerNewUserAccount(UserDto userDto) throws UserAlreadyExistExcep
 
     @Override
     public User deleteUserAccount(String login) {
-        User user = userRepository.findByLogin(login);
-        userRepository.delete(user);
-        return user;
+        User userToDelete = userRepository.findByLogin(login);
+        userRepository.delete(userToDelete);
+        return userToDelete;
+    }
+
+    @Override
+    public User updateUserAccount(String login, UserDto userDto) {
+        return null;
+    }
+
+    @Override
+    public User showUserAccount(String login) {
+        return userRepository.findByLogin(login);
```

Comment:
to samo co w przypadku `deleteUser()` trzeba obsłużyć wyjątek, jeśli nie będzie użytkownika o podanym loginie